### PR TITLE
Basic Design Auth and routing

### PR DESCRIPTION
**Pull request by Developers-codz**
## [Live](https://notesify-git-design-auth-developers-codz.vercel.app/)

### CODE REVIEW

File: src/Redux/Reducers/authSlice.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { AlertToast,SuccessToast } from "../../components/toasts";
+
+const initialState = {
+    currentUser:{},
+};
+
+export const login = createAsyncThunk("/auth/login", async (userDetail) =>{
+    try {
+        const response = await axios.post(`api/auth/login`,JSON.stringify(userDetail));
+        SuccessToast("Login Successful")
```
Comment (vishalg8454) [inline src/Redux/Reducers/authSlice.js:12]: you can move these to extra reducers.

File: src/Redux/Reducers/authSlice.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        return response.data;
+        
+    } catch (error) {
+        console.log(error)
+        AlertToast(`${error}`)
```
Comment (vishalg8454) [inline src/Redux/Reducers/authSlice.js:17]: same

File: src/Redux/Reducers/authSlice.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+export const signup = createAsyncThunk("/auth/signup", async (userDetail) => {
+    try{
+        const response = await axios.post("/api/auth/signup",JSON.stringify(userDetail))
+        SuccessToast("SignIn Successful")
+        return response.data
+    }
+    catch(error){
+        AlertToast(`${error}`)
+
+    }
```
Comment (vishalg8454) [inline src/Redux/Reducers/authSlice.js:30]: you can check https://redux-toolkit.js.org/api/createAsyncThunk#handling-thunk-errors for better error handling with thunk.

File: src/pages/home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+          placeholder="Add title ....."
+          style={{ backgroundColor: bgColor }}
+        />
+        <ReactQuill
+          modules={modules}
+          placeholder={"Add notes......"}
+          height={"100px"}
+        />
+        <EditorFooter>
+        <Pallatte>
```
Comment (vishalg8454) [inline src/pages/home/Home.jsx]: typo in Pallete.

**Review by vishalg8454 — APPROVED** [review]
Good to merge. Left a few suggestions do check them. Also run prettier on the code once before pushing to github.

File: src/Redux/Reducers/authSlice.js (hunk 1 above)
Comment (Developers-codz) [reply src/Redux/Reducers/authSlice.js:12]: Sure

File: src/Redux/Reducers/authSlice.js (hunk 3 above)
Comment (Developers-codz) [reply src/Redux/Reducers/authSlice.js:30]: Yep, actually will do in next pr

File: src/pages/home/Home.jsx (hunk 4 above)
Comment (Developers-codz) [reply src/pages/home/Home.jsx]: okay

File: src/Redux/store.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { configureStore } from "@reduxjs/toolkit";
+import authReducer from "./Reducers/authSlice"
+// import postReducer from "./Reducers/postsSlice";
+// import userReducer from "./Reducers/userSlice"
```
Comment (Dev-AkshatGupta) [inline src/Redux/store.js]: remove the commented code

File: src/components/navbar/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+          <Heading>Notesify</Heading>
+        </Wrapper>
+        <SearchWrapper>
+          <SearchInput placeholder="Search..." />
+
+          <SearchWrapperIcon>
+            <Search />
+          </SearchWrapperIcon>
+        </SearchWrapper>
+
```
Comment (Dev-AkshatGupta) [inline src/components/navbar/Navbar.jsx:32]: Indent this code properly

File: src/index.js [hunk 7]
```diff
@@ -1,15 +1,22 @@
...
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
+    <Provider store={store}>
+    <BrowserRouter>
+
     <App />
+    </BrowserRouter>
```
Comment (Dev-AkshatGupta) [inline src/index.js:18]: Suggestion please keep something in your vscode so that you can indent your code properly

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything seems good to to me just few suggestions 
Else you are good to go

File: src/components/navbar/Navbar.jsx (hunk 6 above)
Comment (Developers-codz) [reply src/components/navbar/Navbar.jsx:32]: yep

File: src/index.js (hunk 7 above)
Comment (Developers-codz) [reply src/index.js:18]: yep, prettier for that

**Pull request merged**